Lombiq.TrainingDemo.csproj

```diff
@@ -23,29 +23,29 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="7.0.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="7.0.0" />
-    <PackageReference Include="OrchardCore.Users.Abstractions" Version="1.7.0" />
-    <PackageReference Include="OrchardCore.Workflows.Abstractions" Version="1.7.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="8.0.0" />
```

Comment:
@Psichorex why not using a .props file to simplify dependencies?

Reply:
Do you mean managing dependencies centrally in the solution? Because these submodules should be possible to build on their own too.